about/people.yml

```diff
@@ -196,14 +120,65 @@ data:
     brown_directory_uuid: e0e55df3-9f2c-4a12-a1de-742955164e1e
```

Comment:
@yangliu2009 is on github, let's add his username

Reply:
Updated !